packages/circuit-ui/package.json

```diff
@@ -72,17 +72,13 @@
     "@types/react-modal": "^3.10.5",
     "component-playground": "^3.2.1",
     "cross-env": "^7.0.2",
-    "css-loader": "^6.7.1",
```

Comment:
These loaders look unused. Are they? Could they have been required by Storybook before?

Reply:
I think they might be a leftover from the static style extraction 🤔